src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -65,6 +65,9 @@ protected void onCreate(Bundle savedInstanceState) {
                     throw new IllegalArgumentException(
                             "You must provide the requested fragment type as an extra.");
             }
+        } else{
+            setTitle(R.string.settings_title);
+            mFragment = new SettingsFragment();
```

Comment:
As stated in the previous PR, this is needed.
Log is an activity with a 'return' behavior to SettingsActivity. Therefore the SettingsActivity will be started with an empty action intent.